[2021-03-22 21:15:06,273] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-03-21T00:00:00+00:00 [queued]>
[2021-03-22 21:15:06,285] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-03-21T00:00:00+00:00 [queued]>
[2021-03-22 21:15:06,286] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:06,286] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-22 21:15:06,286] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:06,294] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_0> on 2021-03-21T00:00:00+00:00
[2021-03-22 21:15:06,298] {standard_task_runner.py:52} INFO - Started process 65 to run task
[2021-03-22 21:15:06,300] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-03-21T00:00:00+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmptv6qohnx', '--error-file', '/tmp/tmpnwmott2b']
[2021-03-22 21:15:06,301] {standard_task_runner.py:77} INFO - Job 10: Subtask runme_0
[2021-03-22 21:15:06,337] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_0 2021-03-21T00:00:00+00:00 [running]> on host 699fbf69fb73
[2021-03-22 21:15:06,374] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-03-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-21T00:00:00+00:00
[2021-03-22 21:15:06,374] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-22 21:15:06,375] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_0__20210321" && sleep 1
[2021-03-22 21:15:06,379] {bash.py:169} INFO - Output:
[2021-03-22 21:15:06,380] {bash.py:173} INFO - example_bash_operator__runme_0__20210321
[2021-03-22 21:15:07,381] {bash.py:177} INFO - Command exited with return code 0
[2021-03-22 21:15:07,403] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20210321T000000, start_date=20210322T211506, end_date=20210322T211507
[2021-03-22 21:15:07,432] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-22 21:15:07,477] {local_task_job.py:146} INFO - Task exited with return code 0
